zeebe/broker-client/src/main/java/io/camunda/zeebe/broker/client/impl/BrokerTopologyManagerImpl.java

```diff
@@ -241,4 +244,19 @@ public void onClusterConfigurationUpdated(final ClusterConfiguration clusterTopo
           }
         });
   }
+
+  private void notifyOnClusterChangeCompleted(
```

Comment:
:upside_down_face: It's not just notifying, it's also updating the `topologyToUpdate`. Maybe `updateOnClusterChangeCompleted` would be clearer?